lib/Conversion/HandshakeToFIRRTL/HandshakeToFIRRTL.cpp

```diff
@@ -543,9 +543,9 @@ static ValueVectorList extractSubfields(FModuleOp subModuleOp,
     auto type = arg.getType().cast<FIRRTLType>();
     if (auto bundleType = type.dyn_cast<BundleType>()) {
       // Extract all subfields of all bundle ports.
-      for (auto &element : bundleType.getElements()) {
-        subfields.push_back(
-            rewriter.create<SubfieldOp>(insertLoc, arg, element.name));
+      for (size_t index = 0, size = bundleType.getNumElements(); index < size;
+           ++index) {
+        subfields.push_back(rewriter.create<SubfieldOp>(insertLoc, arg, index));
```

Comment:
nit: using shorter variable names can save a line:
```suggestion
      for (size_t i = 0, e = bundleType.getNumElements(); i != e; ++i) {
        subfields.push_back(rewriter.create<SubfieldOp>(insertLoc, arg, i));
```